### PR TITLE
Add assigned numbers files

### PR DESCRIPTION
**Pull request by beaufortfrancois**
This PR comes with https://github.com/WebBluetoothCG/web-bluetooth/pull/560

@reillyeon Please review

### CODE REVIEW

File: gatt_assigned_characteristics.txt [hunk 1]
```diff
@@ -0,0 +1,217 @@
+# This file holds a list of human-readable names for standardized GATT
```
Comment (reillyeon) [inline gatt_assigned_characteristics.txt:1]: Here and in the other files can you reference the revision of the Bluetooth Assigned Numbers specification these values are based on?

**Review by reillyeon — CHANGES_REQUESTED** [review]

File: gatt_assigned_characteristics.txt (hunk 1 above)
Comment (beaufortfrancois) [reply gatt_assigned_characteristics.txt:1]: These values are not based on the Bluetooth Assigned Numbers document. They're based on what is currently in Chromium to better reflect what is happening in the real world for now. It was updated last in June 2018. See https://chromium-review.googlesource.com/c/chromium/src/+/1100126/

Note that the Bluetooth Assigned Numbers document latest revision date is `2021-08-25`
As noted earlier, we still need to create a policy to update/remove names.

Comment (reillyeon) [reply gatt_assigned_characteristics.txt:1]: That makes sense to me. Can you note that in these files so it is clear where this data came from?

Comment (beaufortfrancois) [reply gatt_assigned_characteristics.txt:1]: Done in https://github.com/WebBluetoothCG/registries/pull/29/commits/54f04d4b1c25c8394f171418193eeeec4cc35d80

Comment (beaufortfrancois) [reply gatt_assigned_characteristics.txt:1]: If that looks good to you, I'll merge this PR and https://github.com/WebBluetoothCG/web-bluetooth/pull/560

**Review by reillyeon — APPROVED** [review]

File: gatt_assigned_characteristics.txt (hunk 1 above)
Comment (scheib) [reply gatt_assigned_characteristics.txt:1]: I think this text can cite the previous URL https://www.bluetooth.org/en-us/specification/assigned-numbers which was the source of the numbers added to chromium.  The current wording states they are not based on the assigned number document.... but they were at the time they were added to chromium.  I think that should be made clear.

Comment (beaufortfrancois) [reply gatt_assigned_characteristics.txt:1]: Please suggest some new wording and I'll adjust pr tomorrow. Thank you!

File: gatt_assigned_characteristics.txt [hunk 2]
```diff
@@ -0,0 +1,219 @@
...
+# characteristics and their UUIDs to use with the Web Bluetooth API.
+# Those are not based on the Bluetooth Assigned Numbers document. They're based
+# on what is in Chromium as of June 2018.
```
Comment (reillyeon) [inline gatt_assigned_characteristics.txt]:
```suggestion
# These values are based on the mappings implemented by Chromium as of 2018-06-13
# which are based the version of the Bluetooth Assigned Numbers document published
# at that time.
#
# https://source.chromium.org/chromium/chromium/src/+/main:third_party/blink/renderer/modules/bluetooth/bluetooth_uuid.cc;drc=18f27659d87f9464ea5b98022812f054ba5c480b
```

Comment (beaufortfrancois) [reply gatt_assigned_characteristics.txt]: Thank you!

**Pull request merged**